src/Summoner/CLI.hs

```diff
@@ -37,20 +38,30 @@ summon = execParser prsr >>= runWithOptions
 -- | Run 'hs-init' with cli options
 runWithOptions :: InitOpts -> IO ()
 runWithOptions (InitOpts projectName cliConfig maybeFile) = do
-    file <- case maybeFile of
-        Nothing -> (</> "summoner.toml") <$> getHomeDirectory
-        Just x  -> pure x
+    (isDefault, file) <- case maybeFile of
+        Nothing -> (True,) <$> defaultConfigFile
+        Just x  -> pure (False, x)
     isFile <- doesFileExist file
     fileConfig <-
         if isFile
-        then loadFileConfig file
-        else pure mempty
+        then do
+            infoMessage $ "Configurations from " <> T.pack file <> " will be used."
+            loadFileConfig file
+        else  if isDefault
+              then do
+                  warningMessage "Default config file is missing."
```

Comment:
I think you can add path `defaultTomlFile` path to this message just in case. So stupid people won't complain in a way like _It says to me that default config file is missing but I don't know where default config file should be_.